src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```

Comment:
- [ ] it would be better to remove the disable eslint and fix the errors eslint help you to write better and clean code so please try to do that